packages/apps/lightning-ui-docs/.storybook/main.js

```diff
@@ -19,29 +20,24 @@ import remarkGfm from 'remark-gfm'; // needed for Tables, Links, etc in MDX
 const path = require('path');
 
 const config = {
-  addons: [
-    {
-      name: '@storybook/addon-essentials',
-      options: {
-        backgrounds: false,
-        outline: false, // disable outline addon
-        measure: false, // disable measure addon
-        viewport: false // disable viewport addon
-      }
-    },
-    {
-      name: '@storybook/addon-docs',
-      options: {
-        mdxPluginOptions: {
-          mdxCompileOptions: {
-            remarkPlugins: [remarkGfm] //  needed for MDX to use Github Flavored Markdown
-          }
+  addons: [{
+    name: '@storybook/addon-essentials',
+    options: {
+      backgrounds: false,
+      outline: false, // disable outline addon
+      measure: false, // disable measure addon
+      viewport: false // disable viewport addon
+    }
+  }, {
+    name: '@storybook/addon-docs',
+    options: {
+      mdxPluginOptions: {
+        mdxCompileOptions: {
+          remarkPlugins: [remarkGfm] //  needed for MDX to use Github Flavored Markdown
         }
       }
-    },
-    '@storybook/addon-designs',
-    '@storybook/addon-storysource'
-  ],
+    }
+  }, getAbsolutePath("@storybook/addon-designs"), getAbsolutePath("@storybook/addon-storysource"), '@storybook/addon-webpack5-compiler-babel'],
```

Comment:
It gets added automatically to `main.js` as part of the update script, and apparently is standard for Storybook installations in monorepos. It looks to where the package is installed at the top level `package.json` instead of locally.